Adds using to assert_num_queries (#1170)
pytest-dev/pytest-django: docs/helpers.rst

``django_assert_num_queries``
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> .. py:function:: django_assert_num_queries(num, connection=None, info=None, using=None)

Reviewer: ```suggestion
.. py:function:: django_assert_num_queries(num, connection=None, info=None, *, using=None)
```

  :param num: expected number of queries

>   :param connection: optional non-default DB connection
>   :param str info: optional info message to display on failure
>   :param str using: optional the non-default DB connection name

Reviewer 2: "default" would work, right? No need to say "non-default".

```suggestion
  :param connection: optional database connection
  :param str info: optional info message to display on failure
  :param str using: optional database connection name
```

Author: I wanted to keep the changes to a minimal and avoid changing things i also disagreed with, but it’s been like this for a long time; going against the grain is always rougher!

Author: LOL You’re a member. lol my bad!

>   :param str using: optional database connection name

Reviewer: ```suggestion
  :param str using: optional database alias
```

``django_assert_max_num_queries``
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> .. py:function:: django_assert_max_num_queries(num, connection=None, info=None, using=None)

Reviewer: ```suggestion
.. py:function:: django_assert_max_num_queries(num, connection=None, info=None, *, using=None)
```

  :param num: expected maximum number of queries

>   :param connection: optional non-default DB connection
>   :param str info: optional info message to display on failure
>   :param str using: optional the non-default DB connection name

Reviewer 2: ```suggestion
  :param connection: optional database connection
  :param str info: optional info message to display on failure
  :param str using: optional database connection name
```

Author: Ditto.

I’d change this if the maintainer wants it.

>   :param str using: optional database connection name

Reviewer: ```suggestion
  :param str using: optional database alias
```
